core/tests/puppeteer-acceptance-tests/user-utilities/blog-post-admin-utils.js

```diff
@@ -212,6 +236,31 @@ module.exports = class e2eBlogPostAdmin extends baseUser {
     }, blogPostTitle);
   }
 
+  /**
+   * This function checks that the user is unable to publish a blog post.
+   */
+  async expectUserUnableToPublishBlogPost(expectedWarningMessage) {
+    const toastMessageBox = await this.page.$(
+      'div.e2e-test-toast-warning-message');
+    const toastMessageWarning = await this.page.evaluate(
+      el => el.textContent, toastMessageBox);
+    const isPublishButtonDisabled = await this.page.$eval(
+      publishBlogPostButton, (button) => {
+        return button.disabled;
+      });
+
+    if (!isPublishButtonDisabled) {
+      throw new Error('User is able to publish the blog post');
+    }
+    if (expectedWarningMessage !== toastMessageWarning) {
+      throw new Error(
+        'Expected warning message is not same as the actual warning message');
```

Comment:
In the error, say what the expected and actual warning messages are (perhaps in parens). That will help folks debug issues.

Reply:
@seanlip  please check the changes once.

Reply:
Done. Just a reminder, next time, to please assign reviewers if you want a review from them.

Reply:
yeah, sure!